packages/expo-clipboard/ios/Tests/ClipboardModuleSpec.swift

```diff
@@ -0,0 +1,353 @@
+import ExpoModulesTestCore
+
+@testable import ExpoModulesCore
+@testable import ExpoClipboard
+import UIKit
+import MobileCoreServices
+
+class MockUIPasteboard: UIPasteboard {
```

Comment:
Could you move this to the separate file? This class seems to be quite big 😉 
Also please, prettify this code 😛